src/test/CheckResultsByLevel.test.tsx

```diff
@@ -0,0 +1,354 @@
+    expect(anotherLevelHeaderParagraphs.length).toEqual(3);
+    expect(anotherLevelHeaderParagraphs.at(0).text()).toEqual("and another one (0)");
+    expect(anotherLevelHeaderParagraphs.at(1).text()).toEqual("0"); // failed counter
+    expect(anotherLevelHeaderParagraphs.at(2).text()).toEqual("0"); // passed counter
+  });
+    
+  it('renders a failing check appropriately', () => {  
+    const wrapper = getWrapper(checkResultsByLevelData, 4, 2);
+    
+    const checkCandidates = wrapper.find("div#accordion-check-1");
+    expect(checkCandidates.length).toEqual(1);
+    const check = checkCandidates.at(0);
+    expect(check.props().style?.backgroundColor).toEqual("#ff000055");
+    const checkHeader = check.find("p.MuiTypography-root");
+    expect(checkHeader.length).toEqual(1);
+    expect(checkHeader.at(0).text()).toEqual("Status: failed • Service Ownership check");
+    expect(checkHeader.at(0).props().className).toContain("coloredText");
+    const checkSubheader = checkHeader.find("span");
+    expect(checkSubheader.length).toEqual(1);
+    expect(checkSubheader.at(0).text()).toEqual(" • Service Ownership check");
+    expect(checkSubheader.at(0).props().className).toContain("coloredSubtext");
+    const checkContents = check.find("div.MuiAccordionDetails-root");
+    expect(checkContents.length).toEqual(1);
+    expect(checkContents.at(0).find(".p-will-be-enabled").props().hidden).toBe(true);
+    expect(checkContents.at(0).find(".p-check-message").text()).toEqual("Error: This check has status failed.");
+    expect(checkContents.at(0).find(".p-last-updated").props().hidden).toBe(false);
+    expect(checkContents.at(0).find(".p-last-updated").text()).toEqual("Last updated: May 11th 2023, 20:47:53 (UTC)");
+  });
+
+  it('renders a pending check appropriately', () => {  
+    const wrapper = getWrapper(checkResultsByLevelData, 4, 2);
+
+    const checkCandidates = wrapper.find("div#accordion-check-2");
+    expect(checkCandidates.length).toEqual(1);
+    const check = checkCandidates.at(0);
+    expect(check.props().style?.backgroundColor).toEqual("#ffff0055");
+    const checkHeader = check.find("p.MuiTypography-root");
+    expect(checkHeader.length).toEqual(1);
+    expect(checkHeader.at(0).text()).toEqual("Status: pending • Service Ownership check");
+    expect(checkHeader.at(0).props().className).toContain("coloredText");
+    const checkSubheader = checkHeader.find("span");
+    expect(checkSubheader.length).toEqual(1);
+    expect(checkSubheader.at(0).text()).toEqual(" • Service Ownership check");
+    expect(checkSubheader.at(0).props().className).toContain("coloredSubtext");
+    const checkContents = check.find("div.MuiAccordionDetails-root");
+    expect(checkContents.length).toEqual(1);
+    expect(checkContents.at(0).find(".p-will-be-enabled").props().hidden).toBe(true);
+    expect(checkContents.at(0).find(".p-check-message").text()).toEqual("Error: This check has status pending.");
```

Comment:
It shouldn't say Error for pending check results, and it doesn't. I believe this test case may be wrong

<img width="1033" alt="image" src="https://github.com/OpsLevel/backstage-plugin/assets/83714714/7058a5a3-3045-4439-bc63-ddbd6e3c03da">